exercise-src/react-native/17-offline-support/02-solution/src/services/pmo/favorite/hooks.ts

```diff
@@ -1,146 +1,114 @@
 import { useEffect, useState } from "react"
 
-import { storeData, getData } from "../../storage/storage"
+import { getData, storeData } from "../../storage"
 import { apiRequest } from "../api"
 
-interface Favorite {
-  userId: string
-  restaurantId: string
-  favorite: boolean
-  datetimeUpdated: Date
-  _id?: string
-}
-
-interface FavoritesResponse {
-  data: Favorite[] | undefined
-  error: Error | undefined
-  isPending: boolean
-}
-
-interface FavoriteResponse {
-  data: Favorite | undefined
-  error: Error | undefined
-  isPending: boolean
-}
-
-interface LocalStorageFavorites {
-  lastSynced: Date
-  favorites: Favorite[]
-}
+import { Favorite, FavoriteResponse, StoredFavorites } from "./interfaces"
 
 export const useFavorites = (
```

Comment:
Function only allows you to see and affect one favorite, so it should be called `useFavorite`